apps/zotonic_mod_acl_user_groups/src/mod_acl_user_groups.app.src

```diff
@@ -0,0 +1,11 @@
+{application, zotonic_mod_acl_user_groups, [
+    {description, "Zotonic module for organizing users into hierarchical groups"},
+    {vsn, "git"},
+    {registered, []},
+    {applications, [zotonic_core]},
+    {env, []},
+    {modules, []},
+    {maintainers, ["Arjan Scherpenisse <arjan@miraclethings.nl>"]},
+    {licenses, ["Apache 2.0"]},
+    {links, [{"GitHub", "https://github.com/zotonic/zotonic"}]}
+]}.
```

Comment:
This is basically all config we need for publishing to Hex. We will be publishing library OTP applications, so we don’t need a `{mod, …}` callback module.